src/rgw/rgw_lua_background.cc

```diff
@@ -0,0 +1,63 @@
+#include "rgw_lua_background.h"
+#include <lua.hpp>
+
+void Background::set_script(std::string script){
+    rgw_script = script;
+}
+
+bool Background::run(lua_State* L, req_state* s){
```

Comment:
this function should load the script from the object, run it, and sleep for a configurable amount of time
this should happen in an infinite loop guarded with a "continue" boolean variable

please also add a "shutdown()" function to set "continue" to "false" + join on the thread for controller termination fo the thread